docs/content/concepts/partitions-schedules-sensors/partitioning-assets.mdx

```diff
@@ -33,23 +33,6 @@ Before continuing, you should be familiar with:
 
 ---
 
-## Relevant APIs
```

Comment:
Not sure how to comment on this section specifically, but for this tab https://erin-io-doc-114.dagster.dagster-docs.io/concepts/partitions-schedules-sensors/partitioning-assets#managed-loading-dependencies "Managed-loading dependencies" should we add the "Heads up" about I/O manager familiarity? This example uses I/O manager related APIs (`ins`)

Reply:
I can, but I don't think we explicitly state this anywhere. It's kind of implied in the API docs as there's a parameter that mentions I/O managers ([link](https://docs.dagster.io/_apidocs/assets#dagster.AssetIn.metadata)), but that's all I could find. Do we need to add a more explicit explanation somewhere first?